integration-cli/docker_cli_start_test.go

```diff
@@ -47,12 +48,12 @@ func (s *DockerSuite) TestStartAttachCorrectExitCode(c *check.C) {
 	dockerCmd(c, "wait", out)
 
 	startOut, exitCode, err := dockerCmdWithError("start", "-a", out)
-	if err != nil && !strings.Contains("exit status 1", fmt.Sprintf("%s", err)) {
-		c.Fatalf("start command failed unexpectedly with error: %v, output: %q", err, startOut)
-	}
-	if exitCode != 1 {
-		c.Fatalf("start -a did not respond with proper exit code: expected 1, got %d", exitCode)
-	}
+	// start command should fail
+	c.Assert(err, checker.NotNil, check.Commentf("startOut: %s", startOut))
```

Comment:
@HuKeping I don't think so. The previous condition is not that clear but the test should fail, so `NotNil` is alright there :wink:.